user-profile/src/components/CandidateAcademicInfo/CandidateAcademicInfo.test.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import { render, screen, fireEvent, waitFor } from '@testing-library/react';
+import mockData from "./mockData"
+import CandidateAcademicInfo from './CandidateAcademicInfo';
+
+describe('CandidateAcademicInfo component test', () => {
+
+    beforeEach(() => {
+        jest.spyOn(global, 'fetch').mockResolvedValue({
+            json: jest.fn().mockResolvedValue(mockData)
+        })
+    });
+
+    it('should get user data on first render', () => {
+        render(<CandidateAcademicInfo />);
+        expect(fetch).toHaveBeenCalledTimes(1);
```

Comment:
I would also test the the response content from this, so that we can ensure that the mockData is rendered correctly